include/swift/AST/DiagnosticsParse.def

```diff
@@ -1245,6 +1245,22 @@ ERROR(expr_selector_expected_property_expr,PointsToFirstBadToken,
 ERROR(expr_selector_expected_rparen,PointsToFirstBadToken,
       "expected ')' to complete '#selector' expression", ())
 
+// LOLCode expressions.
+ERROR(expected_expr_lolcode,PointsToFirstBadToken,
+      "expected LOLCode expression", ())
+ERROR(expected_of_lolcode_builtin,PointsToFirstBadToken,
+      "expected 'OF' before operands in LOLCode operator", ())
+ERROR(expr_lolcode_expected_lparen,PointsToFirstBadToken,
+      "expected '(' following '#lolcode'", ())
+ERROR(expr_lolcode_expected_rparen,PointsToFirstBadToken,
+      "expected ')' to complete '#lolcode' expression", ())
+ERROR(statement_lolcode_same_line_without_comma,none,
+      "consecutive statements on a line must be separated by ','", ())
+ERROR(statement_lolcode_no_yarly,none,
+      "expected 'YA RLY' statement after 'O RLY?' statement", ())
+ERROR(statement_lolcode_aggressive_orly,none,
+      "'O RLY?' statement must be phrased as a question", ())
```

Comment:
It seems unfortunate that the error messages for LOLCODE ([sic](http://lolcode.org)) are in the same formal tone as the ones for the rest of the compiler. To make this part of the language more compelling, I suggest having form follow…something…and taking inspiration from the syntax itself:

```
ERROR(expected_of_lolcode_builtin,PointsToFirstBadToken,
      "ABSENCE OF 'OF" -> NO COMPILER LOVE", ())

ERROR(statement_lolcode_same_line_without_comma,none,
      "WHY CANT I HOLD ALL THESE STATMENTS", ())
ERROR(statement_lolcode_no_yarly,none,
      "DONT LEAV ME HANGIN", ())
ERROR(statement_lolcode_aggressive_orly,none,
      "Q U E S T I O N   E V E R Y T H I N G", ())
```